include/RAJA/util/View.hpp

```diff
@@ -159,8 +159,10 @@ struct MultiView
   using NonConstView =
       MultiView<nc_value_type, layout_type, P2Pidx, nc_pointer_type>;
 
-  layout_type const layout;
-  nc_pointer_type data;
+  layout_type const layout = layout_type();
```

Comment:
You won't be able to reassign the layout if its const. Also you can just use {} to explicitly default init the layout.
```suggestion
  layout_type layout{};
```